scripts/cli_options.py

```diff
@@ -263,9 +263,9 @@ def get_tumbler_parser():
             action='store',
             dest='makercountrange',
             help=
-            'Input the mean and spread of number of makers to use. e.g. 6 1 will be a normal distribution '
-            'with mean 6 and standard deviation 1 inclusive, default=6 1 (floats are also OK)',
-            default=(6, 1))
+            'Input the mean and spread of number of makers to use. e.g. 9 1 will be a normal distribution '
+            'with mean 9 and standard deviation 1 inclusive, default=9 1 (floats are also OK)',
+            default=(9, 1))
```

Comment:
I do agree, as discussed, but let's be fair, it is arguable.

First, it'll be more expensive on tx fees and Takers already pay a lot there (depending on chain conditions). I do expect people more often to have a liquidity failure, but the code tweaks the schedule down on number of counterparties in some cases and tweaks amounts in others, and of course defaults can be switched.

Anyone else have an opinion?